tests/state/test_state_api.py

```diff
@@ -212,6 +213,14 @@ def test_get_states_page_size(self, size, stub_broker, api_client, state_middlew
         else:
             assert len(res.json["data"]) == size
 
+    def test_not_raise_error_with_pickle_and_non_serializable(
+        self, pickle_encoder, stub_broker, api_client, state_middleware
+    ):
+        state = State("id1", args={"name": "some_name", "date": date(2020, 12, 12)})
+        state_middleware.backend.set_state(state, ttl=1000)
+        res = api_client.get("messages/states")
+        assert res.json == {"count": 1, "data": [state.as_dict(encode_args=True)]}
```

Comment:
write real value instead of `state.as_dict(encode_args=True)`, make thinks more understandable